hydrax/alg_base.py

```diff
@@ -226,17 +286,18 @@ def update_params(self, params: Any, rollouts: Trajectory) -> Any:
         Returns:
             The updated policy parameters.
         """
-        pass
 
-    @abstractmethod
-    def get_action(self, params: Any, t: float) -> jax.Array:
+    def get_action(self, params: SamplingParams, t: jax.Array) -> jax.Array:
         """Get the control action at a given point along the trajectory.
 
         Args:
             params: The policy parameters, U ~ π(params).
-            t: The time (in seconds) from the start of the trajectory.
+            t: The time from the start of the trajectory of shape (1,).
```

Comment:
"The time from the start of the trajectory" is a little confusing to me now that it looks like splines are not defined to start from `t0 = 0` but `t0 = state.time`

(unless I am miss interpreting the implementation, then just ignore)

Reply:
Thanks, there are probably a couple of stale comments throughout. Fixed.